frontend/src/pages/DetailedCourse/DetailedCourse.tsx

```diff
@@ -91,10 +91,10 @@ const DetailedCourse: React.FC<IProps> = ({
           <DetailedCourseTextMobile>{commonCourseData.description}</DetailedCourseTextMobile>
         ) : (
           <DetailedCourseTextMobile>
-            {commonCourseData.description.slice(0, 140)}
-            <ButtonFullText onClick={toggleFullText}>
-              {!isFullTextOpen && OPEN_FULL_TEXT}
-            </ButtonFullText>
+            {commonCourseData.description.slice(0, 140).concat('...')}
+            {Number(commonCourseData.description.length) > 140 && (
+              <ButtonFullText onClick={toggleFullText}>{OPEN_FULL_TEXT}</ButtonFullText>
+            )}
```

Comment:
```suggestion
            const DESCRIPTION_MAX_LENGHTH = 140
            const courseDescription = commonCourseData.description
            const shortedCourseDescriptions = courseDescription.slice(0,DESCRIPTION_MAX_LENGTH).concat('...')
            {shortedCourseDescriptions }
            {courseDescription.length) > DESCRIPTION_MAX_LENGTH && (
              <ButtonFullText onClick={toggleFullText}>{ButtonLabels.openFullText}</ButtonFullText>
            )}
```

Reply:
And what about case, when we have description with length less then 140? We will just add '...'? Its should be improved